client/src/env.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace NodeJS {
+  interface ProcessEnv {
+    NODE_APP_API_URL: string;
+  }
+}
```

Comment:
Did you want to ignore this file? why do you need it? Also its name is a bit strange.

Reply:
Yeah, I was trying to setup .env on reactjs+typescript but is not working